_pages/en_US/troubleshooting.txt

```diff
@@ -127,13 +152,16 @@ Browser based exploits (such as browserhax or 2xrsa) are often unstable and cras
 1. Test booting GodMode9
     1. On Luma3DS, hold (Start) on boot
 1. Try deleting home menu's extdata
-    1. Clear Home Menu's extdata by navigating to the following folder on your SD card: `/Nintendo 3DS/(32 Character ID)/(32 Character ID)/extdata/00000000/`
-        + EUR Region: Delete `00000098`
-        + JPN Region: Delete `00000082`
-        + USA Region: Delete `0000008f`
-        + CHN Region: Delete `000000A1`
-        + KOR Region: Delete `000000A9`
-        + TWN Region: Delete `000000B1`
+    1. Power off your device
+    1. Launch GodMode9 by holding (Start) during boot
+    1. Press (Home) to bring up the action menu
+    1. Select "Scripts..."
+    1. Select "remove_extdata"
+    1. When prompted, press (A) to proceed
+    1. Press (A) to unlock SysNAND (lvl1) writing, then input the key combo given
```

Comment:
The user will not be prompted to unlock SysNAND writing because this is just deleting something from the SD card.